system-parachains/collectives/collectives-polkadot/src/ambassador/migration.rs

```diff
@@ -0,0 +1,389 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Migrations.
+
+use frame_support::{pallet_prelude::*, traits::OnRuntimeUpgrade, weights::Weight};
+
+pub(crate) mod add_accounts {
+	use super::*;
+	#[cfg(feature = "try-runtime")]
+	use alloc::vec::Vec;
+	use frame_support::{parameter_types, traits::RankedMembers};
+	use pallet_ranked_collective::{Config, Pallet as RankedCollective, Rank};
+	#[cfg(feature = "try-runtime")]
+	use pallet_ranked_collective::{MemberCount, Members};
+
+	parameter_types! {
+		// Public key (hex)
+		// This list was created by collating community-submitted addresses from an off-chain document source
+		// https://docs.google.com/spreadsheets/d/1uE5nDKuMZDqlj9q2tvnk_tngyU1Cokl0tQKwSigvJLA/edit?gid=0#gid=0,
+		// then converting each Polkadot SS58 address to its raw public key (32-byte hex) using:
+		// `subkey inspect <SS58_ADDRESS>`
```

Comment:
Please add something like this, i put it into the preimages pallet onchain now.
```pre
A batched call with two remarks has been persisted onto the Collectives Parachchain preimages.
Hash: `0xb6087d77d3b4b5966b9096b459283ea6f5617f4a96d1b4829191439f96fe3c6b` 
Transaction link: https://collectives-polkadot.subscan.io/extrinsic/7035411-2
```

<img width="1448" height="466" alt="Screenshot 2025-08-22 at 13 56 38" src="https://github.com/user-attachments/assets/7cdee4bb-124d-429f-a5e4-74682c0e663f" />